tools/metrics/METADATA

```diff
@@ -0,0 +1,16 @@
+name: "metrics"
+description:
+  "Filtered subtree at tools/metrics."
+
+third_party {
+  url {
+    type: GIT
+    value: "https://chromium.googlesource.com/chromium/src"
+  }
+  version: "114.0.5735.329"
```

Comment:
What's the upstream version here ? Chromium 114 ? 

This seems out of sync with the rest of metrics code, why are jumping that far ahead ?

Reply:
The value of the `version` field is a Chromium version. See for example
https://chromereleases.googleblog.com/2023/06/stable-channel-update-for-desktop_26.html
which mentions `114.0.5735.198`. (We could use that one.)